app/assets/stylesheets/admin/_layout.scss

```diff
@@ -172,7 +172,7 @@ $collapse-tabs-width: 710px;
 }
 
 .masthead-menu-item a {
-  color: $link-colour;
+  color: #054472;
```

Comment:
Could this be made into a variable so it's easier to read

Reply:
Have sorted